src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorFileChangeDetectorTest.cs

```diff
@@ -113,7 +113,7 @@ public void FileSystemWatcher_RazorFileEvent_Background_AddRemoveDoesNotNotify()
 
             // Assert
             fileChangeDetector.BlockNotificationWorkStart.Set();
-            Assert.True(fileChangeDetector.NotifyNotificationNoop.Wait(TimeSpan.FromSeconds(2)));
+            Assert.True(fileChangeDetector.NotifyNotificationNoop.Wait(TimeSpan.FromSeconds(10)));
```

Comment:
Is the plan to run it multiple times in this PR to see if it is flaky? Or do you plan to merge it right away?

Reply:
> Is the plan to run it multiple times in this PR to see if it is flaky? Or do you plan to merge it right away?

The test has a high enough success rate running it on the PR wont be sufficient sadly. Given I can't reproduce the test failure locally my plan was to merge it let the CI do its rounds with it. If it ever fails again it will mean we need to re-evaluate this test entirely because there's something more core going on vs. the Ci just being slow